.github/workflows/pr.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
     strategy:
       matrix:
         java: [ 11 ]
-        image: [ "ubi-quarkus-native-image:22.3-java17"]
+        image: [ "ubi-quarkus-graalvmce-builder-image:22.3-java17"]
```

Comment:
@fedinskiy I follow pattern introduced https://github.com/quarkus-qe/quarkus-test-suite/pull/1004/files, where `ubi-quarkus-native-image` was replace with `ubi-quarkus-graalvmce-builder-image`. Personally I don't know...

Reply:
@rsvoboda in the light of your previous comment[1]: should we use `ubi-quarkus-mandrel-builder-image:22.3-java17` for native everywhere from now on?

[1] https://github.com/quarkus-qe/quarkus-startstop/pull/255#pullrequestreview-1277924931

Reply:
Mandrel one is the default now, it's also used in the PR @michalvavrik is refering to - https://github.com/quarkus-qe/quarkus-test-suite/pull/1004/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L126

`where ubi-quarkus-native-image was replace with ubi-quarkus-graalvmce-builder-image` ... that's true for the matrix where we check both mandrel and graalvm.

In Quarkus upstream there are two changed since Quarkus 2.13:
 - New image names
 - Mandrel builder is the default now

So yes, `ubi-quarkus-mandrel-builder-image` should be used on line 44

Reply:
thank you for explanation, fixed